runelite-client/src/main/java/net/runelite/client/plugins/wintertodt/WintertodtPlugin.java

```diff
@@ -290,23 +291,21 @@ else if (messageNode.getValue().startsWith("The brazier has gone out."))
 		}
 
 		boolean wasInterrupted = false;
-		boolean wasDamaged = false;
 		boolean neverNotify = false;
 
 		switch (interruptType)
 		{
 			case COLD:
 			case BRAZIER:
 			case SNOWFALL:
-				wasDamaged = true;
 
 				// Recolor message for damage notification
 				messageNode.setRuneLiteFormatMessage(ColorUtil.wrapWithColorTag(messageNode.getValue(), config.damageNotificationColor()));
 				chatMessageManager.update(messageNode);
 				client.refreshChat();
 
-				// all actions except woodcutting are interrupted from damage
-				if (currentActivity != WintertodtActivity.WOODCUTTING)
+				// all actions except woodcutting and idle are interrupted from damage
+				if (currentActivity != WintertodtActivity.WOODCUTTING && currentActivity != WintertodtActivity.IDLE)
```

Comment:
Is this a bug fix? Does the release notify you when idle when you get hit?

Reply:
I wasn't sure if it was an oversight or intentional but yes, in release you get a damage notification when idle even when set to "On Action Interrupted".

Reply:
I see you added this part as an independent commit. If the rest of the PR is seen as unnecessary I'm happy to move it to the plugin hub

Reply:
Generally I just like to fix bugs independent of adding new features